static/src/javascripts/projects/common/views/giraffe-header.html

```diff
@@ -0,0 +1,12 @@
+<div class="brand-bar__item brand-bar__item--give-to-the-guardian
+                                    brand-bar__item--has-control
+                                    popup-container has-popup
+                                    hide-until-mobile-landscape">
+    <a href="<%= linkHref %>"
+       id="giraffe__header-contribute-link"
+       data-link-name="give to the Guardian"
```

Comment:
Might be worth making this a bit more descriptive eg: `header : give to the Guardian` to differentiate it from other place